include/vcpkg/base/stringview.h

```diff
@@ -39,23 +33,56 @@ namespace vcpkg
         constexpr size_t size() const noexcept { return m_size; }
         constexpr bool empty() const noexcept { return m_size == 0; }
 
+        // intentionally not provided because this may not be null terminated
+        // constexpr const char* c_str() const
+
         std::string to_string() const;
         void to_string(std::string& out) const;
         explicit operator std::string() const { return to_string(); }
```

Comment:
is there a reason we have this conversion operator? now that we're changing stuff, anyways

Reply:
Yes, the explicit one is needed for transparent containers (like `map<string, value>::insert(StringView, xxx)`) to work.